### PR TITLE
Delete many things

### LINKED ISSUE #450: Provide delete action for all course/session entities
please make sure that action control includes (or form panel includes) option to delete. This is for ALL entities in course:

- [x] Course Objective
- [x] Parent Objective
- [x] cohorts
- [x] Mesh - all levels
- [ ] Learning material - all levels
- [x] Course Director
- [x] Session
- [x] session objective
- [x] session offering
- [x] Session description
- [x] All offering details (Learners, Instructors, Location)

### PR DESCRIPTION
**Pull request by jrjohnson**
Add the ability to delete objectives, learning materials, and session description.

Fixes #450

### CODE REVIEW

File: app/locales/en/translations.js [hunk 1]
```diff
@@ -222,6 +222,7 @@ export default {
     'noSelectedLearnerGroups': 'There are no selected learner groups',
     'instructors': 'Instructors',
     'instructorsManageTitle': 'Manage Instructors',
+    'confirmObjectiveRemoval': 'Are you sure you want to delete this objective?',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dart ajax please translate 'Are you sure you want to delete this objective?'

Comment (saschaben) [inline app/locales/en/translations.js]: FR: "Êtes vous sûr de vouloir supprimer cet objective?"

Comment (dartajax) [inline app/locales/en/translations.js]: ES: "¿Está seguro que desea eliminar este objetivo?"

File: app/locales/en/translations.js [hunk 2]
```diff
@@ -271,6 +273,7 @@ export default {
     'copyrightAgreement': "The file I am attempting to upload is my own or I have express permission to reproduce and/or distribute this item and does not contain any protected health information. My use of this file is in compliance with Government and University policies on copyright and information security and my educational program's guidelines for professional conduct. This file also adheres to the Terms and Conditions for this application.",
     'copyrightRationale':'Copyright Rationale',
     'searchPlaceholder': 'Find Learning Material',
+    'confirmRemoval': 'Are you sure you want to remove this learning material?',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dartajax please translate 'Are you sure you want to remove this learning material?'

**Comment by saschaben** [comment]
FR: "Êtes-vous sûr vous voulez supprimer cette matiére d’étude?"

File: app/locales/en/translations.js (hunk 2 above)
Comment (saschaben) [inline app/locales/en/translations.js]: FR: "Êtes-vous sûr vous voulez supprimer cette matiére d’étude?"

Comment (dartajax) [inline app/locales/en/translations.js]: ES: "¿Está seguro que desea eliminar este material de aprendizaje?"

**Comment by jrjohnson** [comment]
@saschaben please review for function.

**Comment by saschaben** [comment]
typo in french trans. for objective deletion: should be FR: "Êtes vous sûr de vouloir supprimer cet objectif?"

**Comment by saschaben** [comment]
typo in french translation of LM removal: should be FR: "Êtes-vous sûr vous voulez supprimer cette matière d’étude?"

**Comment by saschaben** [comment]
:+1:

**Comment by jrjohnson** [comment]
@saschaben reviewed, for some reason code ninja didn't care.

**Comment by jrjohnson** [comment]
:+1: via Sascha we now have translations

**Comment by stopfstedt** [comment]
click-tested and code reviewed. 
we still got focus switching issues when opening a session objective for editing while the course details are displayed, but apart from that this seems to work.

**Pull request merged**